baobab/bnn_priors/models.py

```diff
@@ -0,0 +1,292 @@
+import numpy as np
+from scipy.special import gamma
+import astropy.units as u
+
+def velocity_dispersion_function_CPV2007(vel_disp_grid):
+    """Evaluate the velocity dispersion function from the fit on SDSS DR6
+    by [1]_ on a provided grid.
+
+    Parameters
+    ----------
+    vel_disp_grid : array-like
+        a grid of velocity dispersion values in km/s
+
+    Returns
+    -------
+    array-like, same shape as `vel_disp_grid`
+        the unnormalized velocity dispersion function
+
+    Note
+    ----
+    The returned function is unnormalized. We use the exact fit values also used in LensPop ([2]_).
```

Comment:
I meant that the returned array is not the actual values of dn/dsigma_V at the values of the `vel_disp_grid` but a scaled version of this, so that the elements sum to 1 because we use it as a PMF from which to draw the velocity dispersion values. I've corrected the docstring.

Reply:
ok, if you state this specifically in the function definition, this would be great, thanks!

Reply:
Included in the function docstring header.